routes/admin.js

```diff
@@ -179,18 +180,28 @@ router.get('/', async (req, res) => {
         return n;
     }));
 
+    // de-dupe notes by id, else will get two on edits
```

Comment:
I don't understand why this is necessary. what is creating the duplicate note?

Reply:
I think I know why --- in createNote, the note is being pushed into the index regardless of whether it is new or an update.   Additions to the index should check for uniqueness. Probably should be done at the storage level.

Reply:
(but that will negate the need to dedupe here)